addon/mixins/model-records/resource-list-controller.js

```diff
@@ -0,0 +1,17 @@
+import Ember from 'ember'
```

Comment:
I think this would make sense to be named as `index-controller-mixin`.  I renamed the ones that pertain to edit/new as `resource-controller-mixin` and `resource-route-mixin`